src/test/java/org/springframework/data/elasticsearch/core/ElasticsearchTemplateTests.java

```diff
@@ -790,26 +790,32 @@ public void shouldSortResultsGivenNullLastSortCriteria() {
 		assertThat(sampleEntities.getContent().get(1).getMessage()).isEqualTo(sampleEntity2.getMessage());
 	}
 
-	@Test // DATAES-467
+	// DATAES-467
+	// DATAES-657
+	@Test
```

Comment:
i think we usually have the `// DATAES_xxx`  comments in the same line as the `@Test` annotation.

Reply:
Done.